src/mastra/tools/evergreen/getTaskWithHistory.ts

```diff
@@ -0,0 +1,114 @@
+import { createTool } from '@mastra/core';
+import { z } from 'zod';
+import { TaskHistoryDirection } from '@/gql/generated/types';
+import getTaskTool from './getTask';
+import getTaskHistoryTool from './getTaskHistory';
+
+enum Requester {
+  AdHoc = 'ad_hoc',
+  GitHubMergeQueue = 'github_merge_request',
+  GitHubPR = 'github_pull_request',
+  GitTag = 'git_tag_request',
+  Gitter = 'gitter_request',
+  Patch = 'patch_request',
+  Trigger = 'trigger_request',
+}
+
+const mainlineRequesters = [
```

Comment:
[nit] Accept it, we've rebranded...

```suggestion
const waterfallRequesters = [
```